### PR TITLE
Scrutinizer Code coverage

### PR DESCRIPTION
**Pull request by john-n-smith**
Ask Travis to generate a code coverage report in clover format and send it to Scrutinizer.

### CODE REVIEW

**Comment by john-n-smith** [comment]
@wpillar Scrutinizer don't yet support our `phpcs` ruleset. As soon as `phpcs` is updated we can easily enable it.

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-  - 7.0
+  - '5.5'
+  - '5.6'
+  - '7.0'
   - hhvm
 
 matrix:
   allow_failures:
-    - php: 7.0
+    - php: '7.0'
```
Comment (wpillar) [inline .travis.yml]: What's with the stringy-ness?

Comment (john-n-smith) [inline .travis.yml]: Saw it in some other examples and made my editor behave.

Comment (wpillar) [inline .travis.yml]: I've not seen it before, all the projects I've seen leave it without the quotes...

Comment (john-n-smith) [inline .travis.yml]: Doesn't make any difference to travis, so happy to return it.

**Comment by john-n-smith** [comment]
@wpillar any other thoughts on this?

**Comment by wpillar** [comment]
@john-n-smith mind if I remove the quotes in the .travis.yml file? I know it's really pedantic, it's setting off my OCD.

**Comment by john-n-smith** [comment]
Not one bit.

**Pull request merged**